src/Component/InformationModal/information/layer/FeaturesInformation.tsx

```diff
@@ -0,0 +1,46 @@
+export const exampleConfig = `
+  {
+    "type": "FeatureCollection",
+    "features": [
+      {
+        "type": "Feature",
+        "geometry": {
+          "type": "Point",
+          "coordinates": [102.0, 0.5]
+        },
+        "properties": {
+          "prop0": "value0"
+        }
+      },
+      {
+        "type": "Feature",
+        "geometry": {
+          "type": "LineString",
+          "coordinates": [
+            [102.0, 0.0], [103.0, 1.0], [104.0, 0.0], [105.0, 1.0]
+          ]
+        },
+        "properties": {
+          "prop0": "value0",
+          "prop1": 0.0
+        }
+      },
+      {
+        "type": "Feature",
+        "geometry": {
+          "type": "Polygon",
+          "coordinates": [
+            [
+              [100.0, 0.0], [101.0, 0.0], [101.0, 1.0],
+              [100.0, 1.0], [100.0, 0.0]
+            ]
+          ]
+        },
+        "properties": {
+          "prop0": "value0",
+          "prop1": { "this": "that" }
+        }
+      }
+    ]
+  }
+`;
```

Comment:
Having the examples included in the code makes it quite hard to override it in projects. Maybe it's easier to include it in external files, e.g. the existing formconfig files? Defaults may be derived from the OpenAPI schema?

Reply:
The examples have been removed. They are now coming through the API (see https://github.com/terrestris/shogun/pull/624 ). It also possible to pass a custom example through the formconfigs.